contents/blog/posthog-vs-heap.md

```diff
@@ -0,0 +1,840 @@
+---
+date: 2023-02-28
+title: PostHog vs Heap
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-mixpanel/posthog-vs-mixpanel.jpeg
+featuredImageType: full
+author: ["andy-vandervell"]
+category: General
+tags:
+  - Comparisons
+  - Product analytics
+  - Session recording
+---
+
+Want to know how PostHog and Heap are different? If you remember nothing else, remember these two points:
+
+1. Heap is a product analytics tool designed for non-technical users.
+
+2. PostHog is an all-in-one platform for product analytics, session recording, feature management, and more, built for engineers and product teams.
+
+In this guide, we'll cover:
+
+- Heap and PostHog's [core features](#core-features)
+- [Product analytics](#product-analytics) features and insight types
+- [Session recording](#session-recording) capability 
+- [Third-party integrations](#integrations), including data export and data import options
+- [Privacy](#privacy-compliance) and [security](#admin--security)
+- [Frequently asked questions](#frequently-asked-questions)
+
+## How is PostHog different to Heap?
+
+### 1. PostHog is an all-in-one platform
+
+Heap, like Amplitude and Mixpanel, focuses mainly on product analytics. That means you need to adopt multiple tools for things like session recording, feature management, and A/B testing. PostHog integrates all these features into one platform, so it becomes your single source of truth. We believe you should own your data, so data exports are free on all tiers.
+
+### 2. PostHog's built for engineers
+
+Anyone can use PostHog – just [ask our marketing team](/blog/posthog-marketing) – but our goal is to [help engineers be better at product](/blog/helping-engineers-to-product), so they spend less time shipping tickets, and more time shipping impactful features. 
```

Comment:
```suggestion
Anyone can use PostHog – just [ask our marketing team](/blog/posthog-marketing) – but our goal is to [help engineers be better at product](/blog/helping-engineers-to-product) by giving them the insights and tools they need to ship impactful features at pace. 
```

@ivanagas I think this lands better. Thoughts?

Reply:
yes

Reply:
The kind of feedback I like to hear 😂